netty/BUILD.bazel

```diff
@@ -20,20 +20,21 @@ java_library(
         "@io_netty_netty_codec_http2//jar",
         "@io_netty_netty_codec_socks//jar",
         "@io_netty_netty_common//jar",
-        "@io_netty_netty_transport_native_unix_common//jar",
         "@io_netty_netty_handler//jar",
         "@io_netty_netty_handler_proxy//jar",
         "@io_netty_netty_resolver//jar",
         "@io_netty_netty_transport//jar",
+        "@io_netty_netty_transport_native_unix_common//jar",
```

Comment:
Moving this from line 23 - is that necessary? What does it achieve?

Reply:
It puts them in alphabetical order which makes maintenance easier.

Reply:
Buildifier did it because it was out-of-order. Somehow it had been missed before.